src/RcppExports.cpp

```diff
@@ -134,15 +134,103 @@ BEGIN_RCPP
     return R_NilValue;
 END_RCPP
 }
+// CLI_SetParamVectorStr
+void CLI_SetParamVectorStr(const std::string& paramName, const std::vector<std::string>& str);
+RcppExport SEXP _RcppMLPACK_CLI_SetParamVectorStr(SEXP paramNameSEXP, SEXP strSEXP) {
+BEGIN_RCPP
+    Rcpp::RNGScope rcpp_rngScope_gen;
```

Comment:
This seems unused; did I overlook something?  It seems to be in the other functions too.  (Again, not an R expert, also not an Rcpp expert :))

Reply:
Machine generated. See top of file.  Ignore macros.

Reply:
Ahhh!!  Right.  I missed that.  Thank you!